src/pages/permissions/panels/EditGroupPermissionsPanel.tsx

```diff
@@ -0,0 +1,284 @@
+          "aria-label": "Delete permission",
+        },
+      ],
+      sortData: {
+        resourceType: permission.entity_type,
+        resource: resourceName.toLowerCase(),
+        entitlement: permission.entitlement,
+      },
+    };
+  });
+
+  return (
+    <>
+      <SidePanel
+        isOverlay
+        loading={false}
+        hasError={false}
+        className="edit-permissions-panel"
+      >
+        <SidePanel.Header>
+          <SidePanel.HeaderTitle>{`Change permissions for ${group?.name}`}</SidePanel.HeaderTitle>
+        </SidePanel.Header>
+        <NotificationRow className="panel-notification" />
+        <SidePanel.Content className="u-no-padding">
+          <ScrollableContainer
+            dependencies={[notify.notification]}
+            belowIds={["panel-footer"]}
+            viewPortHeightLimit={700}
+          >
+            <Card title="Add permissions">
+              <span>
+                Entitlements need to be given in relation to a specific
+                resource. Select the appropriate resource and entitlement below
+                and add it to the list of permissions for this group.
+              </span>
+              <PermissionSelector
+                resourceType={resourceType}
+                resource={resource}
+                entitlement={entitlement}
+                onChangeResourceType={setResourceType}
+                onChangeResource={setResource}
+                onChangeEntitlement={setEntitlement}
+                onAddPermission={handleAddPermission}
+              />
+            </Card>
+            <SearchBox
+              externallyControlled
+              value={search}
+              onChange={setSearch}
+            />
+            {!permissions.length ? (
+              <EmptyState
+                className="empty-state full-width show-left-icon"
+                image={<Icon name="plans" className="empty-state-icon" />}
+                title="No permissions"
+              >
+                <p>Select a permission set above and add to the group</p>
+              </EmptyState>
+            ) : (
+              <ScrollableTable
+                dependencies={[permissions, notify.notification]}
+                tableId="permissions-table"
+                belowIds={["panel-footer"]}
+                viewPortHeightLimit={700}
+              >
+                <MainTable
+                  id="permissions-table"
+                  headers={headers}
+                  sortable
+                  emptyStateMsg={"No permissions match the search criteria."}
+                  rows={rows}
+                />
+              </ScrollableTable>
+            )}
+          </ScrollableContainer>
+        </SidePanel.Content>
+        <SidePanel.Footer className="u-align--right">
+          <Button
+            appearance="base"
+            onClick={closePanel}
+            className="u-no-margin--bottom"
+          >
+            Cancel
+          </Button>
```

Comment:
In addition to this [discussion on confirming](https://github.com/canonical/lxd-ui/pull/694/files/bbb1afc13e4bd8d23676c23992e4504a9e903112#r1574461014) the changes.

I think from the interaction it is not clear, that after adding a new entitlement the user still has to click save changes on the bottom of the panel. Maybe adding a line above the actions that says "x permissions will be modified" in a lighter font colour would clarify this. Similar to the "1 identity will be modified" in the assigning groups/identities panel.

The save button should initially be disabled, when there are no changes.

Reply:
As discussed, next to each permission we will have a modified status. Similar to the edit group/identities panels, we will have copy above the save changes button indicating number of modified permissions.